docs/manuals/source/TasksAndConcepts/Plugins.rst

```diff
@@ -1812,11 +1822,127 @@ Troubleshooting
 '''''''''''''''
 If things don't work as expected, make sure that
 
-- the |fd| (FD) works in general, so that you can make simple file backups and restores
-- the Bareos FD Python plugins work in general, try one of
-  the shipped simple sample plugins
-- Make sure *XtraBackup* works as user root, MySQL access needs to be
-  configured properly
+- the |fd| (FD) works in general, so that you can make simple file backups and restores.
+- the Bareos FD Python plugins work in general, try one of the shipped simple sample plugins.
+- *XtraBackup* works as user root, MySQL access needs to be configured properly.
+
+
+
+.. _mariabackupPlugin:
+.. _backup-MariaDB-mariabackup:
+
+MariaDB mariabackup Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. index::
+   single: Plugin; MariaDB Backup
+   single: mariabackup
+
+This plugin uses native mariabackup tool, to make full and incremental backups of |mariadb| databases.
```

Comment:
```suggestion
This plugin uses the tool :command:`mariabackup`  to make full and incremental backups of |mariadb| databases. :command:`mariabackup` is part of the standard mariadb installation.
```

Reply:
Make sense even if Debian admin will search for a moment as mariabackup is packaged apart